Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN opam depext lwt && opam install lwt alcotest conf-libev
 
 # cache opam install of dependencies
 COPY opam /home/opam/src/datakit/opam
-RUN opam pin add datakit.dev /home/opam/src/datakit -n
```

Comment:
Not sure we want the `#HEAD` here. That means we only test committed changes (the `-k` makes sense though).